docs/source/developers/kernel-manager.md

```diff
@@ -22,9 +22,8 @@ gw_client = GatewayClient.instance()
 gw_client.url = "http://my-gateway-server.com:8888"
 
 client = NotebookClient(nb=test_notebook,
-                        kernel_manager_class=GatewayKernelManager,
-                        kernel_name='my_remote_kernel')
-client.execute()
+                        kernel_manager_class=GatewayKernelManager)
```

Comment:
This is a total nit and nothing you introduced, but, for the sake of consistency, I think it would be good to either join the parameter lines on the constructor here or split the parameters on the `NotebookClient` constructor in the kernel-library.md.  That way, users reading these pages will be able to easily determine that the constructors have the same set of parameters, just different manager class references.  To that point, it may be more clear to join the parameter lines into one.